client/lib/signup/site-type.js

```diff
@@ -55,6 +58,7 @@ export const allSiteTypes = [
 	{
 		id: 4,
 		slug: 'education',
+		icon: 'premium-bundle',
```

Comment:
`premium-bundle` doesn't seem to be a valid icon, as it doesn't correspond to a valid plan slug. Did you mean `value_bundle`?